Updates section on the Quality Criteria (#5148)
GoogleChrome/web.dev: src/site/content/en/progressive-web-apps/using-a-pwa-in-your-android-app/index.md

### Quality Criteria

Web developers should use a Trusted Web Activity when they want to include web content in an
Android app.

Web content in a Trusted Web Activity must meet the PWA installability criteria.

> Additionally, in order to match the behaviour users expect from Android applications,

Reviewer: ```suggestion
Additionally, to match the behavior users expect from Android applications,
```

[Chrome 86 introduced a change][18] where failing to handle the following scenarios is considered a
crash:

  - Failure to verify digital asset links at application launch.
  - Failure to return HTTP 200 for an offline network resource request.

>   - An HTTP 404 or 5xx error in the application.

Author: ```suggestion
  - A navigation request returning an HTTP 404 or 5xx error".
```

>   - An HTTP 404 or 5xx error in the application.

Reviewer: ```suggestion
  - An application resource returning an HTTP 404 or 5xx error.
```

Reviewer: I feel like I need some extra context here. If I didn't correctly infer what's returning 404 and 5xx, come up with your own wording.

Author: How does "A navigation request returning an HTTP 404 or 5xx error" sound?

Reviewer: Sounds great.

Author: updated!

> When one of those scenarios happen in the Trusted Web Activity, it will cause a user visible

Reviewer: ```suggestion
When one of those scenarios happens in the Trusted Web Activity, it causes a user visible
```

crash of the Android application. Check out the [guidance][19] on handling those scenarios in your

> Service Worker.

Reviewer: ```suggestion
service worker.
```
